lesson37/src/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./css/reset.css">
+  <link rel="stylesheet" href="./css/style.css">
+  <script type="module" src="./js/utils/authorizationcheck.js"></script>
+  <script src="./js/drawer-menu.js" defer></script>
+  <script src="./js/login.js" defer></script>
+</head>
+
+<body class="fade-in">
+
+  <main>
+    <header class="header">
+      <div class="header__inner">
+        <button type="button" class="hamburger-button" id="js-drawer-open-button" aria-expanded="false">
+          <span class="hamburger-button__line"></span>
+        </button>
+        <nav class="drawer-menu" id="js-drawer-menu" aria-hidden="true">
+          <ul class="drawer-menu__nav-list">
+            <li class="drawer-menu__nav-item">
+              <a href="../notloginuserpage.html" class="drawer-menu__nav-link js-drawer-menu-nav-link">
+                <span class="drawer-menu__en">top</span>
+                <span class="drawer-menu__jp">トップ</span>
+              </a>
+            </li>
+            <li class="drawer-menu__nav-item">
+              <a href="./index.html" class="drawer-menu__nav-link js-drawer-menu-nav-link">
+                <span class="drawer-menu__en"><img src="./img/login-icon.svg" alt=""
+                    class="drawer-menu__nav-icon">login</span>
+                <span class="drawer-menu__jp">ログイン</span>
+              </a>
+            </li>
+            <li class="drawer-menu__nav-item">
+              <a href="./register/member.html" class="drawer-menu__nav-link js-drawer-menu-nav-link">
+                <span class="drawer-menu__en"><img src="./img/register-icon.svg" alt=""
+                    class="drawer-menu__nav-icon">sign
+                  up</span>
+                <span class="drawer-menu__jp">会員登録</span>
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <div class="transition-loading" id="js-transition-loading">
+      <img src="../img/transition-loading.png" alt="" />
+    </div>
+
+    <div class="login-page">
+      <div class="inner">
+        <div class="login-form">
+          <ul class="tab-menu">
+            <p class="tab-menu__item --selected">ログイン</p>
+            <a href="./register/member.html" class="tab-menu__item">会員登録</a>
+          </ul>
```

Comment:
nits
I noticed that there is no `<li>`.
I was wondering if there was something to add with JS, but I couldn't find it.

Reply:
Thanks for the review.
Great...I didn't notice the mistake.
It is very impressive that you are able to notice such details... Thank you.
[a4b8aef](https://github.com/sae-github/JS_lesson/pull/48/commits/a4b8aef3a647db9b66596a6a651362b8909f50bb)